lib/ansible/modules/utilities/logic/include_vars.py

```diff
@@ -18,9 +18,11 @@
 module: include_vars
 short_description: Load variables from files, dynamically within a task
 description:
-  - Loads variables from a YAML/JSON files dynamically from within a file or from a directory recursively during task
-    runtime. If loading a directory, the files are sorted alphabetically before being loaded.
+  - Loads YAML/JSON variables dynamically from a file or from all files in a directory, recursively, during task runtime.
+  - If loading a directory, the files are sorted alphabetically before being loaded.
   - This module is also supported for Windows targets.
+  - Please note that the use of L(delegate_facts,../user_guide/playbooks_delegate.html#delegated-facts) may be required,
+    depending on your use case, when delegating the include.
```

Comment:
I would rephrase this as 'to assign the variables to a different host than inventory_hostname you need to use both delegate_to and set delegate_facts: True'